include/pagmo/algorithms/mbh.hpp

```diff
@@ -360,13 +358,43 @@ class mbh : public algorithm
         }
         m_perturb = perturb;
     }
+    /// Algorithm's thread safety level.
+    /**
+     * The thread safety of a meta-algorithm is defined by the thread safety of the interal pagmo::algorithm.
+     *
+     * @return the thread safety level of the interal pagmo::algorithm.
+     */
+    thread_safety get_thread_safety() const
+    {
+        return m_algorithm.get_thread_safety();
+    }
+    /// Getter for the inner algorithm
+    /**
+     * Returns a const reference to the inner pagmo::algorithm
+     *
+     * @return a const reference to the inner pagmo::algorithm
+     */
+    const algorithm &get_inner_algorithm() const
+    {
+        return m_algorithm;
+    }
+    /// Getter for the inner algorithm
+    /**
+     * Returns a reference to the inner pagmo::algorithm
+     *
+     * @return a reference to the inner pagmo::algorithm
+     */
+    algorithm &get_inner_algorithm()
+    {
+        return m_algorithm;
+    }
     /// Get log.
     /**
      * A log containing relevant quantities monitoring the last call to mbh::evolve(). Each element of the returned
      * <tt>std::vector</tt> is a mbh::log_line_type containing: \p Fevals, \p Best, \p Violated, <tt>Viol. Norm</tt> and
      * \p Trial as described in mbh::set_verbosity().
      *
-     * @return an <tt> std::vector </tt> of mbh::log_line_type containing the logged values Fevals,
+     * @return an <tt> std::vector </tt> of mbh::log_line_type containing the logged values Fevals, Best,
```

Comment:
I know, I did it in the other pull request using find in files. Doing it here too ... hopefully git will be smart about it